packages/richtext-lexical/src/features/blocks/client/componentInline/index.scss

```diff
@@ -2,22 +2,36 @@
 
 @layer payload-default {
   .LexicalEditorTheme__inlineBlock {
-    @extend %body;
-    @include shadow-sm;
     display: inline-block;
-    margin-right: base(0.2);
```

Comment:
Had to move those styles down to the container. Otherwise, you cannot override them when passing a custom block component, as that only replaces the container.